### PR TITLE
add container (docker/podman) capability for cirq release tag &/or br…

### PR DESCRIPTION
**Pull request by kloodz-mrioux**
- Rework Dockerfile base on python-slim
- Add container_run.sh to start or attach container 
- Add Makefile to build / display container (docker|podman) information
- Add container.md initial documentation

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-ENV LANG en_US.UTF-8  
-ENV LANGUAGE en_US:en  
-ENV LC_ALL en_US.UTF-8 
+# ARG default value is no editor 
+ARG BUILD_CIRQ_EDITOR=""
+#ARG BUILD_CIRQ_EDITOR="emacs vim nano"
 
-# Make python3 default
-RUN rm -f /usr/bin/python && ln -s /usr/bin/python3 /usr/bin/python
+LABEL maintainer="Martial Rioux <martial.rioux@kloodz.com>"
```
Comment (vtomole) [inline Dockerfile]: Will you be maintaining this dockerfile? I think we should leave it to cirq maintainers.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+CONTAINER_RUN=
+
+_set_no_cache:
+	$(eval CONTAINER_BUILD_OPT+=--no-cache)
+
+_set_docker_cmd:
+	$(eval CONTAINER_BUILD=docker build)
+	$(eval CONTAINER_RUN=docker run)
+
+_set_podman_cmd:
```
Comment (vtomole) [inline Makefile]: What's podman? Do you use it often?

File: Dockerfile (hunk 1 above)
Comment (kloodz-mrioux) [reply Dockerfile]: Hi Vtomole, Yes we can change for cirq maintainers I will make the update.

File: Makefile (hunk 2 above)
Comment (kloodz-mrioux) [reply Makefile]: Vtomole, Podman is container engine (https://podman.io/). Avaible from RedHat 8 distribution as primary container engine(https://www.redhat.com/en/blog/working-linux-containers-rhel-8-podman-image-builder-and-web-console). The cost to add the capability to run either podman &/or docker is small. The objectif is to provide both images format to gain an enlarged audience.

File: Dockerfile (hunk 1 above)
Comment (kloodz-mrioux) [reply Dockerfile]: requested change is now in place.

**Comment by vtomole** [comment]
Closing as stale: @kloodz-mrioux Feel free to re-open if you want to continue working on this.

**Pull request closed without merging**